ucs-exporter.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""
+    ucs_exporter.py
+    Exports prometheus consumable metrics for UCSM/UCSC.
+    usage: python3 path/to/ucs_exporter.py -u <user> -p <password> -c <config> -i <inventory>
+"""
+
+import time
+import json
+
+import optparse
+from prometheus_client import CollectorRegistry
+from prometheus_client import start_http_server
+from prometheus_client.core import REGISTRY
+
+from modules.main_collector import MainCollector
+
+
+def get_params():
+    """
+    Argument parser
+    :return: dict of arguments
+    """
+    parser = optparse.OptionParser()
+
+    parser.add_option("-c", "--config", help="", action="store", dest="conf")
+    parser.add_option("-p", "--master-password", help="master password to decrypt mpw", action="store",
+                      dest="master_password")
+    parser.add_option("-i", "--inventory", help="Server list", action="store", dest="inventory")
+    parser.add_option("-u", "--user", help="user used with master password", action="store", dest="user")
+
+    (options, args) = parser.parse_args()
+    print(options)
+    print(args)
+    return options
+
+
+def get_inventory(params):
```

Comment:
this is the part, which needs to be triggered in collect(). Add a helper class and have this called without instantiating.
Usually I would add this in a Base Class, which your MainCollector is inheriting from. Then it would just be a `server_list = self.server_list`.